ui/job-view/PushJobs.jsx

```diff
@@ -230,6 +229,7 @@ export default class PushJobs extends React.Component {
             key={id}
             ref={id}
             refOrder={i}
+            filterPlatformCb={this.filterPlatform}
```

Comment:
Need to pass this callback down to each job so that when its selected state changes, it can check if the platform should still be visible (it may have been the last visible job in the platform under the current filters).